### DISCUSSION TITLE
Thanks for this :-)

### DISCUSSION THREAD

**Discussion by florimondmanca** [vkurko/calendar#190] [Show and tell]
Hi there,

I wanted to say thanks for the work on this lightweight version of FullCalendar. I was looking for a more lightweight alternative with a similar look and feel, and this came out almost a drop-in replacement! The calendar bundle size for the calendar part in our app went down from ~ 220 kB to ~ 80 kB.

Here's the commit where this is located: https://github.com/fairnesscoop/permacoop/pull/406/commits/171e869314dda079af1875e5fd4e983a9d999d1e

Cheers.

**Comment by vkurko** [comment]
I'm glad that Event Calendar has become useful in your project.

Small bundle size is one of my priorities. I hope that I can continue to maintain this size, and ideally even make it even smaller :smile:.
